src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+.calendar__day {
+  width: $day-size;
+  height: $day-size;
+  border: $border-size solid black;
```

Comment:
The variable `$border-size` is used here but is not defined in this file. Make sure `$border-size` is defined to avoid compilation errors.